src/pcrlock/pcrlock.c

```diff
@@ -2999,7 +3055,7 @@ static int verb_lock_secureboot_authority(int argc, char *argv[], void *userdata
          * checks, and that's what we should bind policy to. Moreover it's hard to predict, since extension
          * card firmware validation will result in additional records here. */
 
-        if (!is_efi_secure_boot()) {
+        if (!is_efi_secure_boot() && !arg_force) {
```

Comment:
there is no authority though if SB is off. hence there's not uch point in generating this

Reply:
I guess the current behavior is how it is because `systemd-pcrlock-secureboot-authority.service` always calls `lock-secureboot-authority`. If the user toggles secure-boot, pcrlock automatically adds or removes the pcrlock file.
An alternative might be to add `ConditionSecurity=uefi-secureboot` to the service file. But then the pcrlock file wouldn't be removed automatically when secure-boot gets turned off.

Reply:
> there is no authority though if SB is off. hence there's not uch point in generating this

I should be missing something, but if I want to generate a policy that can express "... and secure boot is disabled" I need to measure all the events affecting PCR 7 (policy and authority) as both events are registered in any case in the event log.

For example, I have an EV_EFI_VARIABLE_AUTHORITY for SbatLevel in a disabled SB system.

Reply:
I forgot to mention, but the `--force` is a bit in conflict with: https://github.com/systemd/systemd/pull/30997

Should I revert 30997 and use `--force` instead?